src/reso/agent/ResoAgent.php

```diff
@@ -183,4 +184,44 @@ protected function copyOjoFindings($linkedFiles,$uploadId)
     }
     return true;
   }
-}
+
+  /**
+   * @brief Copy copyright from .license file to base file
+   * @param array[][] $linkedFiles - multi-dimensional array with copyright holding files with associated base files
+   * @param int $uploadId
+   * @return true on success, Error on failure
+   */
+  protected function copyCopyrightFindings($linkedFiles,$uploadId)
+  {
+    //find agentId used for specific upload
+    $latestCopyrightAgent=$this->agentDao->agentARSList("copyright_ars",$uploadId);
+    $resoAgentId=$this->agentDao->getCurrentAgentId("reso");
+    
+    foreach ($linkedFiles as $file) {
+      $this->heartbeat(1);
+      $param = array();
+      $insertParam = array();
+      $stmt = __METHOD__ .'readCopyrightFindings';
+      $sql = "SELECT * FROM copyright WHERE pfile_fk =$1 AND agent_fk=$2";
+      $param[] = $file['pfile_fk'];
+      $param[] = $latestCopyrightAgent[0]['agent_fk'];
+      
+      $this->dbManager->prepare($stmt, $sql);
+      $res = $this->dbManager->execute($stmt,$param);
+      while ($row=$this->dbManager->fetchArray($res)) {
+        $insertParam = array();
+        $Istmt = __METHOD__ .'insertCopyrightFindingsReso';
+        $Isql = "INSERT INTO copyright(agent_fk, pfile_fk, content, hash, type, copy_startbyte, copy_endbyte) VALUES ($1, $2, $3, md5($3), $4, $5, $6)";
```

Comment:
Please set `copy_startbyte` and `copy_endbyte` as `NULL` instead of `-1, -1`

```suggestion
        $Isql = "INSERT INTO copyright(agent_fk, pfile_fk, content, hash, type, copy_startbyte, copy_endbyte) VALUES ($1, $2, $3, md5($3), $4, NULL, NULL)";
```